docs/dev/i18n.rst

```diff
@@ -5,7 +5,10 @@ General guidelines
 ------------------
 
 * All config mode packages must be fully translatable, with complete English and German texts.
-* All new expert mode packages be fully translatable. English texts are required, German texts recommended.
+* All new expert mode packages must be fully translatable. English texts are required.
+* Translations for German tests are recommended. Other supported languages, especially french are
```

Comment:
Typo: "tests", also the sentence doesn't make sense like this. "German translations are recommended" would be clearer.

"French" must be capitalized. A comma is missing after "French".